content/sensu-go/5.11/getting-started/tutorial.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Interactive Tutorial"
+description: ""
+version: "5.11"
+weight: 3
+product: "Sensu Go"
+layout: "katacoda-wide"
+menu:
+  sensu-go-5.11:
+    parent: getting-started
+---
+
+<script src="//katacoda.com/embed.js"></script>
+<div id="katacoda-scenario-1"
+    data-katacoda-id="sensu/sandbox"
+    data-katacoda-color="2c3458"
+    style="height: 600px; padding-top: 20px;"></div>
```

Comment:
Would it be reasonable to make the height auto with a 600px minimum? Currently feels a bit cramped in my browser.

Reply:
I wasn't able to get this to work. Using `auto` or `100%` did not produce the desired behavior. I've updated the height to 800px, which looks decent on both a large monitor and a small screen in fullscreen. We can likely achieve a more resposive behavior using Katacoda's [external style sheet feature](https://dashboard.katacoda.com/docs/embed/parameters)